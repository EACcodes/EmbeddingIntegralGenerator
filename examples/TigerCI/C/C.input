NUM THREADS
16
XYZ FILE
C.xyz
GRID DIMENSIONS
100 80 120
LATTICE VECTORS
10.116199999999999     -10.116199999999999       0.000000000000005
-0.000000000000000      10.116200000000001     -10.116199999999999
9.819982691896255       9.819982691896257       9.819982691896257
OUTPUT FORMAT
TIGERCI
BASIS FILE
basis_set_TZ
END OF INPUT
