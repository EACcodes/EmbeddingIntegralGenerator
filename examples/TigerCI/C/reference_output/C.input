NUM THREADS
16
BASIS SET
cc-pVTZ
XYZ FILE
C.xyz
GRID DIMENSIONS
100 80 120
LATTICE VECTORS
10.116199999999999     -10.116199999999999       0.000000000000005
0.000000000000000      10.116200000000001     -10.116199999999999
9.819982691896255       9.819982691896257       9.819982691896257
OUTPUT FORMAT
TIGERCI
END OF INPUT
