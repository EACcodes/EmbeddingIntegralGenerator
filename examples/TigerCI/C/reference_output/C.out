
********************************************************************************
********************************************************************************
**********                                                            **********
                        EMBEDDING INTEGRAL GENERATOR                            
**********                                                            **********
********************************************************************************
********************************************************************************

********************************************************************************
**********                                                            **********
                        CONFIGURATION OF THIS CALCULATION                       

Keyword                                 Value
-------                                 -----
BASIS SET                               cc-pVTZ
CARTESIAN                               0
EMBEDDING INTS REAL                     0
EMBEDDING POTENTIAL FILE                embpot.dat
GRID DIMENSIONS                         100  80  120  
LATTICE VECTORS                         10.1162  -10.1162  5e-15  
                                        0  10.1162  -10.1162  
                                        9.81998  9.81998  9.81998  
NUM THREADS                             16
OUTPUT FORMAT                           TIGERCI
SHIFT VECTOR                            Not defined, use default:
                                        0  0  0  
XYZ FILE                                C.xyz
**********                                                            **********
********************************************************************************

Running with OpenMP using 16 threads
***********************
*                      
* Embedding Integrals  
*                      
***********************

  Read grid data and embedding potential wall time(s): 0.419 cpu time(s): 6.26

* Reciprocal space integration scheme
  Calculate embedding integrals, real space part wall time(s): 2.648 cpu time(s): 25.67
  Calculate embedding integrals wall time(s): 4.887 cpu time(s): 52.24

TOTAL EXECUTION TIME  wall time(s): 5.374 cpu time(s): 58.86
